contributing/labeling.md

```diff
@@ -0,0 +1,15 @@
+# Label Definitions
+
+* `Epic`: Big issue to be broken down into smaller tasks
+
+### Priority (`p.*`)
+* `p.Critical`: To be handled as soon as possible, usually a bug or security loophole
+* `p.High`: Issues corresponding to the upcoming `Milestone`
+* `p.Medium`: Issues corresponding to a future `Milestone`
+* `p.Low`: Good-to-have features not in the roadmap
+
+### Difficulty (`d.*`)
```

Comment:
don't you think we could go easier on the amount/granularity of labels? imo `FirstTimer/Beginner` is enough as a label. I imagine it being difficult to differentiate Intermediate, Advanced.